src/main/res/values/attrs.xml

```diff
@@ -12,4 +12,10 @@
         <attr name="floatLabelHint" format="reference|string"/>
     </declare-styleable>
 
+    <declare-styleable name="LoadingImageView">
+        <attr name="imageSrc" format="reference" />
+        <attr name="imageHeight" format="reference|dimension"/>
```

Comment:
i don't think you need the `reference` part. It will automatically be a reference by ALSO being a dimension. If you look at how the Android SDK sets attributes for ViewGroup.Layout, you'll see it only needs `dimension`:


```
  <declare-styleable name="ViewGroup_Layout">
        <!-- Specifies the basic width of the view.  This is a required attribute
             for any view inside of a containing layout manager.  Its value may
             be a dimension (such as "12dip") for a constant width or one of
             the special constants. -->
        <attr name="layout_width" format="dimension">
            <!-- The view should be as big as its parent (minus padding).
                 This constant is deprecated starting from API Level 8 and
                 is replaced by {@code match_parent}. -->
            <enum name="fill_parent" value="-1" />
            <!-- The view should be as big as its parent (minus padding).
                 Introduced in API Level 8. -->
            <enum name="match_parent" value="-1" />
            <!-- The view should be only big enough to enclose its content (plus padding). -->
            <enum name="wrap_content" value="-2" />
        </attr>

        <!-- Specifies the basic height of the view.  This is a required attribute
             for any view inside of a containing layout manager.  Its value may
             be a dimension (such as "12dip") for a constant height or one of
             the special constants. -->
        <attr name="layout_height" format="dimension">
            <!-- The view should be as big as its parent (minus padding).
                 This constant is deprecated starting from API Level 8 and
                 is replaced by {@code match_parent}. -->
            <enum name="fill_parent" value="-1" />
            <!-- The view should be as big as its parent (minus padding).
                 Introduced in API Level 8. -->
            <enum name="match_parent" value="-1" />
            <!-- The view should be only big enough to enclose its content (plus padding). -->
            <enum name="wrap_content" value="-2" />
        </attr>
    </declare-styleable>
```

see `/Users/YOUR_USER/Library/Android/sdk/platforms/android-25/data/res/values/attrs.xml`